src/seedu/addressbook/data/AddressBook.java

```diff
@@ -44,6 +47,7 @@ public AddressBook(UniquePersonList persons, UniqueTagList tags) {
         for (Person p : allPersons) {
             syncTagsWithMasterList(p);
         }
+        this.allTaggings = new ArrayList<Tagging>();
```

Comment:
Try to be consistent with your style. Is there any particular reason why this constructor uses `this.allTaggings` while the other uses `allTaggings`?